SPECS/krb5/krb5.spec

```diff
@@ -3,16 +3,20 @@
 
 Summary:        The Kerberos newtork authentication system
 Name:           krb5
-Version:        1.21.3
-Release:        2%{?dist}
+Epoch:          1
```

Comment:
The `%{name} = %{version}-%{release}` dependency in the `devel` and `lang` subpackages would now also need to be changed to `%{name} = %{epoch}:%{version}-%{release}`. Same thing applies for all other packages with a dependency on a specific version of `krb5`.

Reply:
Updated the devel, lang, and lib lines. Please let me know if I missed anything.

Reply:
This spec looks good. Please also check other specs for dependencies (both `Requires` and `BuildRequires`) on a specific version of `krb5` and its packages. I think I haven't seen any but please verify my findings.

Reply:
I also checked in the codebase and no dependency is specifying specifically a krb5 version, only `>= {krb5-version}` so I think we're good on that.